pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/PersistentTopicsTest.java

```diff
@@ -568,4 +568,42 @@ public void testGetLastMessageId() throws Exception {
         Assert.assertTrue(admin.topics().getLastMessageId(topicName) instanceof MessageIdImpl);
 
     }
+
+    @Test()
```

Comment:
turn `@Test()` into `@Test` if no parameter will add to it?

Reply:
updated